package/gluon-core/files/usr/bin/get-mesh-ifs

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+ubus call network.interface dump | jsonfilter -e "@.interface[@.proto='gluon_mesh' && @.up=true].device"
```

Comment:
Maybe name it "gluon-mesh-interfaces"? In any case, it should start with "gluon-".

Reply:
sure. I like verbs, so I would go with gluon-get-mesh-interfaces ok?

Reply:
Ok